### DISCUSSION TITLE
Comparing temperature measurement with slices between simulation

### DISCUSSION THREAD

**Discussion by LordMicidial** [firemodels/fds#13659] [Q&A]
Hi all,

Due to the fact I've been asked to study the egress on a FDS scenario I delivered to competent authority in June. I needed to adjust the fds file in order to run on a machine working with FDS+EVAC. 

I'm here not to seek help on the EVACuation case which I've already understood and addressed with competent authority but to seek help on the FDS computation.

I was happy to see that the model with EVAC which is still running as I'm writing is outputting the .hrr with values that are almost the same of the FDS simulation that I ended in June.  Based on that I was confident that no major mistakes were made on the FDS side.

However while I compare the .smv views on the same slices for Temperature I got significant errors between the values I expected to be the same. Here you see the screenshot.

**Please note that i introduced major changes in the roof openings**.

I also attach the two fds codes.
[240625_DCI1.fds.txt](https://github.com/user-attachments/files/17578520/240625_DCI1.fds.txt)
[241028_DCI1.fds.txt](https://github.com/user-attachments/files/17578522/241028_DCI1.fds.txt)

Can you tell if the only reason why i got different temperatures could be the roof openings ?

MESHes should be equally spaced. hrr. files are almost the same.

Is it legit to state that without openings temperature decreases?

![image](https://github.com/user-attachments/assets/ce1e7922-ef23-4556-b9a1-143848134f0d)

**Comment by Er9y714** [comment]
Assuming these two files are identical and have been run with same version of FDS, different temperatures at a single timestep are possible in transient LES models. Even if you run the same file twice you might encounter this. Also `NOISE` parameter might deviate the results slightly. 

I would compare the average value of DEVCs at the same location instead of looking at slice files.

**Reply by Vojta-Salek** [reply]
I haven't checked the FDS files. However, just based on the two provided pictures - you have different ranges of your temperatures (19.99 to 215.1 °C vs 19.99 to 368 °C) and therefore also different colors for possibly the same temperatures. Apparently your first simulation was terminated earlier than the second one, thus global maximum range of temperature in the second (longer) simulation can be different (higher in your case). 

In Smokeview, you can do Load/Unload ->Slice -> Settings... -> Bound/Truncate data -> and here change the maximum in order to make your colorbar comparable between the two simulations. Anyway, DEVCs are always better for inspecting any particular outputs in detail as suggested above.

**Reply by LordMicidial** [reply]
Second simulation is still running. Thank you for your time.

**Reply by Vojta-Salek** [reply]
Oh, my bad - it is still running, not terminated. However the outcome is still the same - the global maximums are different between the simulations and therefore also the colobars. Of course my point is related only to visualization, I'm not saying anything about the influence of changes in roof openings.
